FernFlower-Patches/0021-Add-a-metadata-file-named-fernflower_abstract_parame.patch

```diff
@@ -104,7 +104,7 @@ index cc4e357fb8b7adeeb76858813f5ee329d17497d5..68cf259230cd82298c36b391e0ea91d1
    }
 +
 +  public void loadAbstractMetadata(String string) {
-+    for (String line : string.split("\n")) {
++    for (String line : string.split(DecompilerContext.getNewLineSeparator())) {
 +      String[] pts = line.split(" ");
```

Comment:
Naw, this *should* be a \n separated file. Unless your data is wrong.
However, this is not how you read files. The OS of the consumer has nothing to do with the OS of the producer.
So better would be to split on "\n" as is done now, and then trim any '\r's that exist at the end.

Reply:
I got it. Because there's no doc for that file, I previously thought that the newline character should follow the system.

Reply:
Since `String.split` takes a Regex pattern anyways, would it make sense to use `\R` (or `\r?\n|\r`; `\r` for old Mac OS) then?